melodic/distribution.yaml

```diff
@@ -12645,7 +12645,7 @@ repositories:
       tags:
         release: release/melodic/{package}/{version}
       url: https://github.com/septentrio-users/septentrio_gnss_driver-release.git
-      version: 1.0.6-1
+      version: 1.0.8-1
```

Comment:
I don't see 1.0.8-1 in the release repo. Did bloom work correctly?

https://github.com/septentrio-users/septentrio_gnss_driver-release/tree/master